RevenueCatUI/Views/IconView.swift

```diff
@@ -35,7 +35,7 @@ struct IconView<S: ShapeStyle>: View {
 /// An icon to be displayed by `IconView`.
 enum PaywallIcon: String, CaseIterable {
 
-    case add
+    case plus
```

Comment:
talk about driving me NUTS. having it named `add` creates a collision with a UIKit name, but the collision happens when Swift auto-generates intermediate files, and has a terrible error message.